features/controller_specs/README.md

```diff
@@ -60,3 +60,21 @@ To specify outcomes, you can use:
 * by default, views are not rendered. See
   [views are stubbed by default](controller-specs/views-are-stubbed-by-default) and
   [render_views](controller-specs/render-views) for details.
+
+## Headers
+
+We encourage you to use [request specs](https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec) if you want
```

Comment:
As from that old issue I remember some were mentioning an alternative approach:
```ruby
post path, params: params.to_json, headers: headers
```
However, since controller specs are deprecated, maybe it is not worth to mention it.

Reply:
I tried it but `headers` was not recognized.

```ruby
require "rails_helper"

RSpec.describe ApplicationController, :type => :controller do
  controller do
    def show
      if request.headers['Authorization'] == 'foo'
        head :ok
      else
        head :forbidden
      end
    end
  end

  before do
    routes.draw { get "show" => "anonymous#show" }
  end

  context "valid Token header" do
    it  do
      puts "Rails version is : #{Rails.version}\n"
      get :show, params: {}, headers: { 'Authorization': 'foo' }
      expect(response).to have_http_status(:ok)
    end
  end

  context "invalid Token header" do
    it do
      get :show
      expect(response).to have_http_status(:forbidden)
    end
  end
end

```

```
Failures:

  1) ApplicationController valid Token header
     Failure/Error: get :show, params: {}, headers: { 'Authorization': 'foo' }

     ArgumentError:
       unknown keyword: headers
     # ./spec/controllers/cookies_controller_spec.rb:21:in `block (3 levels) in <top (required)>'
```

Reply:
Same with Rails 6.0.1